### ISSUE TITLE
For `password input` component, initial focus is not correct if use of `Voice Over` or `Full Keyboard Access`

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1533]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### Device OS version

26

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [ ] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

2.0.0

### Describe the issue

When I launch the `password input` screen, the focus is put on hide password button.
Same thing if I use the keyboard, when using TAB key, the focus goes from top-navbar (theme button) to hide password button

### Expected Behavior

Focus should be set on password input first

### Accessibility settings

VO or keyboard

**Comment by pylapp** [comment]
> [!NOTE]
> Fixed with **046d94201ba348440d25afd02a65c0f9352c110d** (tag **3.0.0-beta.5)**
> because the `OUDSPasswordInput` is based on the `OUDSTextInput`.
> Thus #1562 solves this #1533
